modules/rules_synology/0.0.2/presubmit.yml

```diff
@@ -0,0 +1,14 @@
+# We recommend included a bcr test workspace that exercises your ruleset with bzlmod.
+# For an example, see https://github.com/aspect-build/bazel-lib/tree/main/e2e/bzlmod.
+bcr_test_module:
+  module_path: "examples/cross-helloworld-no-go"
+  matrix:
+    platform: ["ubuntu2204", "ubuntu2404"]
```

Comment:
Unfortunately, ubuntu2404 is not yet supported on Bazel CI..

Reply:
should be resolved in [ec0b616](https://github.com/bazelbuild/bazel-central-registry/pull/2913/commits/ec0b616aeece319bfce35c8274e2c98c43359470)